websites/N/NekoAnime/presence.ts

```diff
@@ -0,0 +1,151 @@
+import { ActivityType, Assets, getTimestampsFromMedia } from 'premid'
+
+const presence = new Presence({
+  clientId: '1379070900653260840',
+})
+
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+
+enum ActivityAssets {
+  Logo = 'https://i.imgur.com/kUavF8n.png',
+}
+
+async function videoActive() {
+  const videoUrl3 = document.querySelector('#MediaPage > div > div.head_player > div.vid-player')
+  if (videoUrl3) {
+    return true
+  }
+  else {
+    return false
+  }
+}
+
+async function getAnimeInformation() {
+  const animeTitle = document.querySelector('#MediaPage > div > div.head_player > div.player_content > div.episode_info > h2')?.textContent
+  const episodeNumber = document.querySelector('#number')?.textContent
+
+  const currentTime = document.querySelector('.vjs-current-time-display')?.textContent
+  const totalTime = document.querySelector('.vjs-duration-display')?.textContent
+
+  const playButton = document.querySelector('.vjs-play-control')
+
+  let videoStatus = ''
+  if (playButton?.classList.contains('vjs-playing')) {
+    videoStatus = 'Reproduciendo'
+  }
+  else if (playButton?.classList.contains('vjs-paused')) {
+    videoStatus = 'Pausado'
+  }
+  else {
+    videoStatus = 'Desconocido'
+  }
+
+  const urlTitulo = document.querySelector('#MediaPage > div > div.head_player > div.player_content > div.anime_navigation > nav > a:nth-child(3)')?.getAttribute('href')
+  const res = await fetch(`https://nekoanime.mx${urlTitulo}`)
+  const html = await res.text()
+  const parser = new DOMParser()
+  const doc = parser.parseFromString(html, 'text/html')
+  const img = doc.querySelector('#InfoApp > div.anime_cont > div > div.left_cont > div.anim-cover > div > figure > img')?.getAttribute('src')
```

Comment:
This code has the potential of sending several requests per minute to the service's server. Add a cache to prevent excessive requests